Resources/config/services.xml

```diff
@@ -11,14 +11,18 @@
         <service id="massive_pdf.pdf_manager" class="%massive_pdf.pdf_manager.class%">
             <argument type="service" id="templating" />
             <argument type="service" id="knp_snappy.pdf" />
+
+            <deprecated>The "%service_id%" service is deprecated use massive_pdf.pdf_factory instead.</deprecated>
```

Comment:
The class was deprecated some time ago so the service definition should also be deprecated.